stdlib/Test/test/runtests.jl

```diff
@@ -110,10 +110,22 @@ let fails = @testset NoThrowTestSet begin
         # Fail - function keyword splatting
         k = [(:atol, 0), (:nans, true)]
         @test isapprox(1, 2; k...)
+        # Fail - isempty
+        nonempty = [1, 2, 3]
+        @test isempty(nonempty)
+        str1 = "Hello"
+        str2 = "World"
+        # Fail - occursin
+        @test occursin(str1, str2)
+        # Fail - startswith
+        @test startswith(str1, str2)
+        # Fail - endswith
```

Comment:
Since this list has now hit 19 elements long,
maybe we should and in these commands the index that this one is causing to fail?

Or perhaps work out another way to refact it,
To move the trigger rand the evaluation closer together.